apollo-router/src/configuration/snapshots/apollo_router__configuration__tests__schema_generation.snap

```diff
@@ -3562,6 +3562,32 @@ expression: "&schema"
       },
       "type": "object"
     },
+    "IntrospectionMode": {
+      "description": "Which implementation of GraphQL schema introspection to use, if enabled",
+      "oneOf": [
+        {
+          "description": "Use the new Rust-based implementation.",
+          "enum": [
+            "new"
+          ],
+          "type": "string"
+        },
+        {
+          "description": "Use the old JavaScript-based implementation.",
+          "enum": [
+            "legacy"
+          ],
+          "type": "string"
+        },
+        {
+          "description": "Use Rust-based and Javascript-based implementations side by side, logging warnings if the implementations disagree.",
+          "enum": [
+            "both"
+          ],
+          "type": "string"
+        }
+      ]
+    },
```

Comment:
Why is it part of this PR ?

Reply:
So we kept having CI failures because of changes to the JSON schema for the config I think? And I thought this would fix it but it doesn't! Any ideas much appreciated!